daml/Tests/BinaryOption.daml

```diff
@@ -46,7 +50,7 @@ endToEnd = script do
       issuanceId = "Myissuance"
       accountId = Id (Set.fromList [alice.customer, providers.bank]) alice.mainAccount.id.label 0
       assetId = optionDesc.assetId
-      quantity = 1000.0
+      quantity = 1000.0 -- TODO is this unused ???
```

Comment:
Apparently this number does not influence anything, is this a bug?

Reply:
This is used to create the `Issuance` contract. That is, how many options are issued and created with the `AssetDeposit`. It is different from the payoff (which is 1$).

Reply:
It still seems surprising that bob had 1000 binary options, they are _all_ exercised, and he gets only $1.
Then they shouldn't be all exercised perhaps?

Reply:
Yes, I see what you mean now. I made a minor change on this branch to fix it. I hope you don't mind.